daprdocs/content/en/developing-applications/building-blocks/state-management/howto-get-save-state.md

```diff
@@ -72,38 +72,27 @@ The following example shows how to save and retrieve a single key/value pair usi
 
 ```csharp
 
-//dependencies
-using System;
-using System.Collections.Generic;
-using System.Net.Http;
-using System.Net.Http.Headers;
+using System.Text;
 using System.Threading.Tasks;
 using Dapr.Client;
-using Microsoft.AspNetCore.Mvc;
-using System.Threading;
-using System.Text.Json;
 
-//code
-namespace EventService
+var builder = WebApplication.CreateBuilder(args);
+builder.Serivces.AddDaprClient();
```

Comment:
```suggestion
builder.Services.AddDaprClient();
```

Reply:
I would add this for completeness 
`    var result = await client.GetStateAsync<string>(DAPR_STORE_NAME, "order_2");`

Also this line seems to have been dropped
`const string DAPR_STORE_NAME = "statestore";`